<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="2" total="10" passed="6" failed="2" skipped="0">
  <reporter-output>
    <line>
      <![CDATA[<a href= "C:\Users\apoorv\OneDrive\Pictures\Screenshots\sel.png">Screenshot link</>]]>
    </line>
    <line>
      <![CDATA[<a href= "C:\Users\apoorv\OneDrive\Pictures\Screenshots\sel.png">Screenshot link</>]]>
    </line>
  </reporter-output>
  <suite started-at="2022-06-12T18:05:04 IST" name="Suite" finished-at="2022-06-12T18:05:05 IST" duration-ms="221">
    <groups>
      <group name="functional">
        <method signature="Dependencies.doUserReg()[pri:1, instance:testCases.Dependencies@49ec71f8]" name="doUserReg" class="testCases.Dependencies"/>
        <method signature="Dependencies.ThirdTest()[pri:3, instance:testCases.Dependencies@49ec71f8]" name="ThirdTest" class="testCases.Dependencies"/>
        <method signature="Dependencies.doLogin()[pri:2, instance:testCases.Dependencies@49ec71f8]" name="doLogin" class="testCases.Dependencies"/>
        <method signature="assertionsCheck.validateTitles()[pri:0, instance:testCases.assertionsCheck@1a482e36]" name="validateTitles" class="testCases.assertionsCheck"/>
      </group> <!-- functional -->
      <group name="smoke">
        <method signature="Dependencies.ThirdTest()[pri:3, instance:testCases.Dependencies@49ec71f8]" name="ThirdTest" class="testCases.Dependencies"/>
        <method signature="Dependencies.FourthTest()[pri:4, instance:testCases.Dependencies@49ec71f8]" name="FourthTest" class="testCases.Dependencies"/>
      </group> <!-- smoke -->
      <group name="sanity">
        <method signature="TestCases.doLogin()[pri:2, instance:testCases.TestCases@15ff3e9e]" name="doLogin" class="testCases.TestCases"/>
      </group> <!-- sanity -->
    </groups>
    <test started-at="2022-06-12T18:05:04 IST" name="basic Test" finished-at="2022-06-12T18:05:04 IST" duration-ms="103">
      <class name="testCases.TestCases">
        <test-method is-config="true" signature="creatingDbConnection()[pri:0, instance:testCases.TestCases@15ff3e9e]" started-at="2022-06-12T18:05:04 IST" name="creatingDbConnection" finished-at="2022-06-12T18:05:04 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- creatingDbConnection -->
        <test-method is-config="true" signature="setup()[pri:0, instance:testCases.TestCases@15ff3e9e]" started-at="2022-06-12T18:05:04 IST" name="setup" finished-at="2022-06-12T18:05:04 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="doUserReg()[pri:1, instance:testCases.TestCases@15ff3e9e]" started-at="2022-06-12T18:05:04 IST" name="doUserReg" finished-at="2022-06-12T18:05:04 IST" duration-ms="5" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doUserReg -->
        <test-method is-config="true" signature="TearDown()[pri:0, instance:testCases.TestCases@15ff3e9e]" started-at="2022-06-12T18:05:04 IST" name="TearDown" finished-at="2022-06-12T18:05:04 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TearDown -->
        <test-method is-config="true" signature="setup()[pri:0, instance:testCases.TestCases@15ff3e9e]" started-at="2022-06-12T18:05:04 IST" name="setup" finished-at="2022-06-12T18:05:04 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="doLogin()[pri:2, instance:testCases.TestCases@15ff3e9e]" started-at="2022-06-12T18:05:04 IST" name="doLogin" finished-at="2022-06-12T18:05:04 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doLogin -->
        <test-method is-config="true" signature="TearDown()[pri:0, instance:testCases.TestCases@15ff3e9e]" started-at="2022-06-12T18:05:04 IST" name="TearDown" finished-at="2022-06-12T18:05:04 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TearDown -->
        <test-method is-config="true" signature="closingDbConnection()[pri:0, instance:testCases.TestCases@15ff3e9e]" started-at="2022-06-12T18:05:04 IST" name="closingDbConnection" finished-at="2022-06-12T18:05:04 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closingDbConnection -->
      </class> <!-- testCases.TestCases -->
    </test> <!-- basic Test -->
    <test started-at="2022-06-12T18:05:04 IST" name="Dependencies test" finished-at="2022-06-12T18:05:04 IST" duration-ms="9">
      <class name="testCases.Dependencies">
        <test-method signature="doUserReg()[pri:1, instance:testCases.Dependencies@49ec71f8]" started-at="2022-06-12T18:05:04 IST" name="doUserReg" finished-at="2022-06-12T18:05:04 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doUserReg -->
        <test-method signature="ThirdTest()[pri:3, instance:testCases.Dependencies@49ec71f8]" started-at="2022-06-12T18:05:04 IST" name="ThirdTest" finished-at="2022-06-12T18:05:04 IST" depends-on-methods="testCases.Dependencies.doUserReg" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ThirdTest -->
        <test-method signature="doLogin()[pri:2, instance:testCases.Dependencies@49ec71f8]" started-at="2022-06-12T18:05:04 IST" name="doLogin" finished-at="2022-06-12T18:05:04 IST" depends-on-methods="testCases.Dependencies.ThirdTest, testCases.Dependencies.doUserReg" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doLogin -->
        <test-method signature="FourthTest()[pri:4, instance:testCases.Dependencies@49ec71f8]" started-at="2022-06-12T18:05:04 IST" name="FourthTest" finished-at="2022-06-12T18:05:04 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- FourthTest -->
      </class> <!-- testCases.Dependencies -->
    </test> <!-- Dependencies test -->
    <test started-at="2022-06-12T18:05:04 IST" name="Failure Test" finished-at="2022-06-12T18:05:05 IST" duration-ms="27">
      <class name="testCases.assertionsCheck">
        <test-method is-config="true" signature="setup()[pri:0, instance:testCases.assertionsCheck@1a482e36]" started-at="2022-06-12T18:05:04 IST" name="setup" finished-at="2022-06-12T18:05:04 IST" duration-ms="20" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="validateTitles()[pri:0, instance:testCases.assertionsCheck@1a482e36]" started-at="2022-06-12T18:05:04 IST" name="validateTitles" finished-at="2022-06-12T18:05:04 IST" duration-ms="14" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[not found this thing expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: not found this thing expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertTrue(Assert.java:56)
at testCases.assertionsCheck.validateTitles(assertionsCheck.java:15)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[<a href= "C:\Users\apoorv\OneDrive\Pictures\Screenshots\sel.png">Screenshot link</>]]>
            </line>
          </reporter-output>
        </test-method> <!-- validateTitles -->
        <test-method is-config="true" signature="TearDown()[pri:0, instance:testCases.assertionsCheck@1a482e36]" started-at="2022-06-12T18:05:05 IST" name="TearDown" finished-at="2022-06-12T18:05:05 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TearDown -->
      </class> <!-- testCases.assertionsCheck -->
      <class name="testCases.SoftyAssert">
        <test-method signature="validateTitles()[pri:0, instance:testCases.SoftyAssert@72967906]" started-at="2022-06-12T18:05:05 IST" name="validateTitles" finished-at="2022-06-12T18:05:05 IST" duration-ms="5" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[cannot register user expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: cannot register user expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:976)
at testCases.SoftyAssert.validateTitles(SoftyAssert.java:16)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[<a href= "C:\Users\apoorv\OneDrive\Pictures\Screenshots\sel.png">Screenshot link</>]]>
            </line>
          </reporter-output>
        </test-method> <!-- validateTitles -->
      </class> <!-- testCases.SoftyAssert -->
    </test> <!-- Failure Test -->
  </suite> <!-- Suite -->
</testng-results>
